src/pages/index.js

```diff
@@ -101,4 +198,19 @@ const profileEditValidator = new FormValidator({
   config: config,
 });
 profileEditValidator.enableValidation();
+api
+  .fetchInitialData()
+  .then(([userData, cardsData]) => {
+    console.log("hey");
+    userInfo.setUserInfo(userData.name, userData.about);
```

Comment:
* When I reload the page, the old `avatar` appears again

You should pass only `userData` here

```js
    userInfo.setUserInfo(userData);


```